Dockerfile

```diff
@@ -0,0 +1,8 @@
+  FROM ruby:3.1.2
+  RUN apt-get update && apt-get install -y nodejs
+  WORKDIR /app
+  COPY Gemfile* .
+  RUN bundle install
+  COPY . .
+  EXPOSE 3000
+  CMD ["rails", "server", "-b", "0.0.0.0"]
```

Comment:
should we be running bin/dev? otherwise, does tailwind work just by running rails s?
i assume if we do a prebundle it will work fine?